src/RegistrationController.js

```diff
@@ -172,6 +184,9 @@ function (
       // register parse complete event listener
       self.state.get('schema').on('parseComplete', function(updatedSchema) {
         var modelProperties = updatedSchema.properties.createModelProperties();
+        if (!updatedSchema.error) {
+          self.settings.set('policyId', updatedSchema.properties.policyId);
```

Comment:
Are we trying to overwrite the `policyId` when a default `policyId` is present here ?

It feels like we should create another variable called `defaultPolicyId` in Settings.js to keep them separate and set it here when we want to use the default `policyId`. Also for this case I feel it would be ideal if the API actually sent the key as `defaultPolicyId`.